DEVELOPING.md

```diff
@@ -31,7 +23,9 @@ The command will produce the `revive` binary in the root of the project.
 
 If you want to develop a new rule, follow as an example the already existing rules in the [rule package](https://github.com/mgechev/revive/tree/master/rule).
 
-All rules should implement the following interface:
+## Writing a Custom Rule
+
+Each rule needs to implement the `lint.Rule` interface:
```

Comment:
Please add info about the [`lint.ConfigurableRule`](https://github.com/mgechev/revive/blob/7d97f5481a4c07ed7b492718abec69c2bb71932f/lint/rule.go#L21) interface. All rules with a configuration must implement that interface.